### PR TITLE
Add the option to impose a minimal width in RegionGeom.to_wcs_geom()

### PR DESCRIPTION
**Pull request by LauraOlivera**
**Description**

This pull request adds the option `width_min` to the existing method `RegionGeom.to_wcs_geom()` to force a minimum width for the geometry. The input can be both a single value or 2, one for each spatial dimension

### CODE REVIEW

File: gammapy/maps/region.py [hunk 1]
```diff
@@ -185,17 +185,33 @@ def bin_volume(self):
...
+            Minimal width for the resulting geometry.
+            Can be a single number or two, for 
+            different minimum widths in each spatial dimension.
+
         Returns
         -------
         wcs_geom : `~WcsGeom`
             A WCS geometry object.
         """
+        if width_min is not None:
```
Comment (adonath) [inline gammapy/maps/region.py:204]: Does this work?
```python3
width = np.min([self.width.to_value("deg"), _check_width(width_min)], axis=0)
```
Where `_check_width()` is imported from `gammapy.maps.wcs`...

**Review by adonath — COMMENTED** [review]
Thanks @LauraOlivera, I have left two inline comments...

File: gammapy/maps/region.py [hunk 2]
```diff
@@ -185,17 +185,33 @@ def bin_volume(self):
...
         return bin_volume
 
-    def to_wcs_geom(self):
+    def to_wcs_geom(self, width_min=None):
         """Get the minimal equivalent geometry that
             contains the region.
 
+        Parameters
+         ----------
+            width_min : `~astropy.quantity.Quantity`
```
[the remark this answers is not in the document]
Comment (LauraOlivera) [reply gammapy/maps/region.py:194]: ups

File: gammapy/maps/region.py (hunk 1 above)
Comment (LauraOlivera) [reply gammapy/maps/region.py:204]: oh yes, that is fantastic! once again, had no idea this existed, very useful function!

**Review by adonath — APPROVED** [review]
Thanks @LauraOlivera, no further comments from my side...

**Pull request merged**